source_modelling/srf.py

```diff
@@ -230,37 +230,24 @@ def planes(self) -> list[Plane]:
         """list[Plane]: The list of planes in the SRF."""
```

Comment:
Additional comment to clarify why this isn't as straightfoward as one might assume.